http/src/test/java/org/mule/test/module/http/functional/listener/HttpListenerLifecycleTestCase.java

```diff
@@ -44,15 +45,15 @@ protected String getConfigFile() {
 
   @Test
   public void stopOneListenerDoesNotAffectAnother() throws Exception {
-    ExtensionMessageSource httpListener = (ExtensionMessageSource) ((Flow) getFlowConstruct("testPathFlow")).getSource();
+    ExtensionComponent httpListener = (ExtensionComponent) ((Flow) getFlowConstruct("testPathFlow")).getSource();
```

Comment:
This was the only place outside extension support modules referencing ExtensionMessageSource, but it is not required to reference it as the only used method exists on ExtensionComponent.
It could be debated why if ExtensionComponent is API then why ExtensionMessageSource is not... I prefer to expose the minimum number of classes, unless you disagree